boostrap-3/style.scss

```
@@ -0,0 +1,438 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box !important;
+}
+body {
+    background-color: #4e5366;
+
+    .container {
+
+        .container__header {
+            ul {
+                list-style: none;
+                display: flex;
+                li{
```

```suggestion
                li {
```
Trước dấu `{` hãy thêm 1 dấu cách